packages/client/src/tests/schema.graphql

```diff
@@ -422,6 +432,46 @@ type CertificateSVG {
   status: String
 }
 
+type Question {
+  _id: ID
+  fieldId: String
```

Comment:
These two should be required fields no? And also I don't think we have an `id` field in Question schema now do we?

Reply:
_Id always auto-added in mongo schema. And yes `fieldId` should be required for input and type.